integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdown.java

```diff
@@ -453,7 +459,7 @@ public void testAddShutdownOptionsToMsTimeout() throws Exception {
       updateCrdYamlVerifyShutdown(crd, delayTime);
       Assertions.assertTrue(
           checkShutdownUpdatedProp(domainUid + "-managed-server1", "20", "false", "Graceful"));
-      if (terminationTime > (3 * 20 * 1000)) {
+      if (terminationTime > (4 * 20 * 1000)) {
```

Comment:
magic number?

Reply:
it cause some Jen runs to fail intermittently due some env issues, so I increased total time to avoid it ( will review this test in other pr)